Aidan/dind+bugfix (#233)
latchbio/latch: docs/source/subcommands.md

Generate a Dockerfile for the workflow instead of relying on [auto-generation.](basics/defining_environment.md#automatic-dockerfile-generation)

#### `--base-image`, `-b`

> One of `opencl`, `cuda`, `docker`. If provided, the workflow will be built using with the specified software in the environment. These options are mutually exclusive.

Reviewer: also have a `base` option to get the normal one

Reviewer: e.g. '`base`, `opencl`, `cuda`, `docker`, defaulting to `base` if not provided'

Reviewer: or ig `default` judging by the `init` function

> One of `opencl`, `cuda`, `docker`, `default`. If provided, the workflow will be built using with the specified software in the environment. These options are mutually exclusive. If not provided or `default`, will use a bare-minimum image.

Reviewer 2: - `If provided` is the most redundant thing ever
- `specialized` is a weird value judgement
- use the explanation that I wrote in `docs/source/basics/defining_environment.md`, it's more descriptive than what's here
- ideally even delete it from that file to avoid redundancy and just leave a link to here
```suggestion
One of `opencl`, `cuda`, `docker`, or `default`. INSERT EXPLANATION FROM THE OTHER FILE
```
